setup.cfg

```diff
@@ -90,8 +90,7 @@ test =
     smart-open >= 5.0.0
     urllib3 >= 1.26.5
 spark =
-    pyspark >=3.2.0
-    pandas >=1.3.0, <1.4.0
+    woodwork[spark] >= 0.16.2
```

Comment:
Since woodwork 0.16.x and 0.17.x have incompatible spark requirements and featuretools still allows woodwork 0.16.x I tied the spark requirements to woodwork's spark requirements